Update cluster.rst (#4024)
mattermost/docs: source/deployment/cluster.rst

Deployment Guide
----------------

> Deployment guide to set up and maintain High Availability on your Mattermost servers. This document does not cover the configuration of databases in terms of disaster recovery, however, please refer to the `Frequently Asked Questions (FAQ)`_ section for our recommendations.

Reviewer: ```suggestion
Deployment guide to set up and maintain High Availability on your Mattermost servers. This document does not cover the configuration of databases in terms of disaster recovery, however, you can refer to the `Frequently Asked Questions (FAQ)`_ section for our recommendations.
```

Plugins and High Availability
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. File storage is assumed to be shared between all the machines that are using services such as NAS or Amazon S3.

Reviewer 2: ```suggestion
As of Mattermost 5.14, when you install or upgrade a plugin, it is propagated across the servers in the cluster automatically. File storage is assumed to be shared between all the servers, using services such as NAS or Amazon S3.
```

Current wording doesn't parse.

Yes. Although not officially tested, you can set up a cluster across AWS regions, for example, and it should work without issues.

What does Mattermost recommend for diaster recovery of the databases?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When deploying Mattermost in a High Availability configuration, we recommend using a database load balancer between Mattermost and your database. Depending on your deployment this needs more or less consideration.

Reviewer: Do we mean depending on the size of the deployment, your choice of load balancer will differ?

Author: @justinegeffen Do you think removing this sentence would make it hard to transition to the two following statements? I was trying to come up with a better transition but was finding it challenging.

Reviewer: I like the intention of the statement, and thinking something like this could work:

"The load balancer configuration depends on the scale of your deployment, so requirements will differ."

> For example, if deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple availability zones.

Reviewer: ```suggestion
For example, if deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple Availability Zones.
```

When deploying Mattermost on your own cluster please refer to your IT team for a solution best suited for your existing architecture.

> 

Reviewer: 3/5: The two sentences in one paragraph with a minor rewording:
"For example, if you're deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple Availability Zones. If you're deploying Mattermost on your own cluster please consult with your IT team for a solution best suited for your existing architecture."

Yes. Although not officially tested, you can set up a cluster across AWS regions, for example, and it should work without issues.

What does Mattermost recommend for diaster recovery of the databases?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

When deploying Mattermost in a High Availability configuration, we recommend using a database load balancer between Mattermost and your database. Depending on your deployment this needs more or less consideration.

> For example, if you're deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple Availability Zones. If you're deploying Mattermost on your own cluster please consult with IT team for a solution best suited for your existing architecture.

Reviewer 2: ```suggestion
For example, if you're deploying Mattermost on AWS with Amazon Aurora we recommend utilizing multiple Availability Zones. If you're deploying Mattermost on your own cluster please consult with your IT team for a solution best suited for your existing architecture.
```

Not sure who `IT Team` is - I assume there's a `your` missing there?
